template/{% if use_git and dev_platform == 'GitHub' %}.github{% endif %}/workflows/release.yml

```diff
@@ -1,53 +1,74 @@
-name: CD
+name: Publish to PyPI
 
 on:
-  push:
-    tags:
-    - 'v?[0-9]+.[0-9]+.[0-9]+'
-    - 'v?[0-9]+.[0-9]+.[0-9]+(a|b|rc|post|dev)[0-9]+'
-
-jobs:
-  prerequisites:
-    uses: ./.github/workflows/test.yml
-
   release:
-    needs: [prerequisites]
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        python-version: ["3.12"]
-    runs-on: ${{ matrix.os }}
-    permissions:
-      # Write permissions are needed to create OIDC tokens.
-      id-token: write
-      # Write permissions are needed to make GitHub releases.
-      contents: write
+    types: [published]
 
+jobs:
+  # Setup build separate from publish for added security
+  # See https://github.com/pypa/gh-action-pypi-publish/issues/217#issuecomment-1965727093
+  build:
+    runs-on: ubuntu-latest
+    # This ensures that the publish action only runs in the main repository
+    # rather than forks
```

Comment:
I assume that an `if:` condition was probably removed from here, and the comment is a remnant?

Reply:
oooh yes likely that was the case!! i put the publish step into it's own independent step for security!